website/docs/reference/widgets/table/inline-editing.md

```diff
@@ -1,134 +1,109 @@
-
-```js
-{{ currentRow.status === "pending" }}
-```
-Inline editing does not save changes automatically to the original datasource. To save changes, the table's **Update mode** must be set and **onSave** event must be configured to execute queries that update the datasource.
-
 
-### Update single row
-
-To update a single row, select **Single row**  from the **Update mode** property. When one or more columns of a table are **Editable** in Single row mode, a new table column is inserted called **Save/Discard**. This new column contains a Save button and a Discard button, which execute the table's **onSave** and **onDiscard** events when clicked. 
+## Enable inline editing 
 
+Enable inline editing by checking the **Editable** checkbox for the desired columns in the Table widget properties panel. Once inline editing is enabled, you can edit cell contents by clicking on the ✏️ icon or double-clicking on a cell.
 
+<figure>
+  <img src="/img/inline-22.png" style= {{width:"600px", height:"auto"}} alt="Display images on table row selection"/>
+  <figcaption align = "center"><i>Enable inline editing</i></figcaption>
+</figure>
 
----
-**Example**: suppose you want to modify the `name`  and `phone` field in a `users` database.
 
-1. Fetch data from the sample **users** database using a SELECT query `fetchData` to retrieve the data.
+## Update single row
 
-```sql
-SELECT * FROM users ORDER BY id LIMIT 10;
-```
+To update a single row, follow these steps:
 
-2. In the Table's **Table Data** property, display the data using:
 
+1). Select **Single row**  from the **Update mode** property. This enables the **Save/Discard** column with Save and Discard buttons.
 
-```js
-{{fetchData.data}}
-```
+2). Create a new query that uses the `updatedRow` reference property to update the relevant data.
 
-3. Select the **Editable** checkbox for the `name` and `phone` column properties.
+<dd>
 
-4. Create a new query that uses the `updatedRow` reference property.
+_Example_: Suppose you want to modify the `name`  and `phone` field in a `users` database.
 
 ```sql
 UPDATE users SET 
   name = {{Table1.updatedRow.name}},
-	phone = {{Table1.updatedRow.phone}}
+  phone = {{Table1.updatedRow.phone}}
   WHERE id = {{ Table1.updatedRow.id }};
 ```
 
-5. Set the Table widget's **onSave** event to run the update query, and **onSuccess** callback to run `fetchData` query.
-
-As an alternative you can configure the **onSubmit** event in each [column's settings](/reference/widgets/table/column-settings) to run a query that saves the new data. The **onSubmit** event takes place whenever the user clicks away from the edited cell, or presses the Enter key within it. 
-
+</dd>
 
+3). Click on the gear icon ⚙️ next to the **Save/Discard** column in the property pane.
 
-### Update multiple rows
+4). Set the **onSave** event to run the update query, and **onSuccess** callback to trigger a fetch query that refreshes the data.
```

Comment:
```suggestion
4). Set the **onSave** event to run the update query, and the  **onSuccess** callback to trigger the fetch query that refreshes the table data with the updated information.
```

Reply:
repeat this wherever the same statement is being used

Reply:
done